src/vaults/mainnet.json

```diff
@@ -550,6 +550,11 @@
       "slug": "kodiak-yeet-wbera",
       "stakeTokenAddress": "0xec8ba456b4e009408d0776cde8b91f8717d13fa1"
     },
+    {
+      "beraRewardsVault": "0x39b08cCbdfD9B0D3C45F1FD02E9b57B44C88ed71",
+      "slug": "osbgt-ewbera-4",
+      "stakeTokenAddress": "0x0B77545A4EB1B995e79BF85e0ed8d6Ee288ca632"
```

Comment:
`0x0B77545A4EB1B995e79BF85e0ed8d6Ee288ca632` does not exist in our tokens. Should this be `0xCaa70d2aa873Ef057980844e18D9a9560bdfFcC0`, or something else?